src/ui/public/agg_types/metrics/percentile_ranks.js

```diff
@@ -47,6 +47,8 @@ define(function (require) {
         return fieldFormats.getInstance('percent') || fieldFormats.getDefaultInstance('number');
       },
       getValue: function (agg, bucket) {
+        if (!bucket[agg.parentId]) return;
```

Comment:
What is the purpose of this? I see it repeated in a few aggs. Is this needed in all multi-metric aggs?